### DISCUSSION TITLE
Can't access Hot Reload due to Proxy

### DISCUSSION THREAD

**Discussion by sasakrsmanovic** [unoplatform/studio#51] [Q&A] [answered]
### Current behavior

Incoming email:
We are using Uno Platform to develop a mobile application. But noticed that we can't use Hot Reload because we can't authorize due to Proxy.
Which servers need to be added to the restrictions for it to start working? Or is it possible without authorization?

### Which tool(s) are affected?

Hot Reload

**Comment by dr1rrb** [comment, marked as answer]
We will add proper doc for this, but for ref, answer to the original question is:
* https://platform.uno/ (Including from a web browser to allow sign-in)
* https://licencing.api.platform.uno/

**Comment by dr1rrb** [comment]
@sasakrsmanovic
